### PR TITLE
feat: Zoom Meeting Added

### LINKED ISSUE #623: Integrate Zoom with Meetups
## Description
As a meetup creator, I want to able to create zoom links in the UI itself for virtual meetups.

## Definition of Done
- [x] All of the required items are completed.
- [x] Approval by 2 mentor.

## Estimation
120 hours

### PR DESCRIPTION
**Pull request by ritwickraj78**
### Description
Include a summary of the change and relevant motivation/context. List any dependencies that are required for this change.

Fixes #623 

### Type of Change:
**Delete irrelevant options.**

- Code

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by sammy1997 — DISMISSED** [review]
LGTM! @sakshi1499 @SanketDG?

**Review by sakshi1499 — COMMENTED** [review]
I think Sanket and I were planning to test this one before merging! Other than that, I feel the docstring "date less than the current date" could be "date prior to the current date".

File: systers_portal/meetup/views.py
Comment (SanketDG) [inline systers_portal/meetup/views.py]: For places where you have directly accessed the `dict`, I would check for `KeyError`, since you are directly using the data from Zoom.

If it's KeyError, send back an HTTP 500 or something.

File: systers_portal/meetup/utils.py
Comment (SanketDG) [inline systers_portal/meetup/utils.py]: feel free to remove those

**Review by SanketDG — CHANGES_REQUESTED** [review]
Excellent work from a first look :rocket:

File: systers_portal/meetup/views.py
Comment (ritwickraj78) [reply systers_portal/meetup/views.py]: Done

File: systers_portal/meetup/utils.py
Comment (ritwickraj78) [reply systers_portal/meetup/utils.py]: Done

**Comment by ritwickraj78** [comment]
@SanketDG @sakshi1499 @sammy1997 This one is gtg.

File: systers_portal/meetup/views.py
Comment (SanketDG) [inline systers_portal/meetup/views.py:133]: could have just used if else here, no?

File: systers_portal/meetup/utils.py
Comment (SanketDG) [inline systers_portal/meetup/utils.py:195]: could have refactored this in a single method.

**Review by SanketDG — APPROVED** [review]
LGTM, feel free to merge this, if the comments don't get resolved, I will open them as github issues since they are minor anyway.

**Review by sammy1997 — APPROVED** [review]
I agree with the issues pointed out by @SanketDG. Other than that, LTGM!

**Comment by sammy1997** [comment]
@sakshi1499 can you review and merge the 2 PRs? This one and the pin board one?

**Review by sakshi1499 — APPROVED** [review]
LGTM! Merging this one!

**Pull request merged**
